salt/utils/win_dacl.py

```diff
@@ -825,8 +826,14 @@ def list_aces(self):
                 ace = self.dacl.GetAce(i)
 
                 # Get ACE Elements
-                user, a_type, a_prop, a_perms, inh = self._ace_to_dict(ace)
-
+                try:
+                    user, a_type, a_prop, a_perms, inh = self._ace_to_dict(ace)
+                except pywintypes.error as exc:
+                    # If at this point a mapping error is thrown 
+                    # we should just skip the entry, instead of failing because of this
+                    if exc.winerror == winerror.ERROR_NONE_MAPPED:
+                        continue
+                    raise
                 # Check for existing entries in the return
```

Comment:
We need to return the SID if there is no mapping, not skip. We need to see the actual permissions for the object.